config/routes.rb

```diff
@@ -28,12 +28,12 @@
       # Not in V1
       #has_logic
 
-      resources :questions, only: [] do
-        resources :community_solutions, shallow: true, except: [:index] do
-          publishable
+      resources :questions, only: []# do
+        #resources :community_solutions, shallow: true, except: :index do
+          #publishable
           #has_logic
-        end
-      end
+        #end
+      #end
```

Comment:
commented out?

Reply:
I didn't want us to forget how it was setup if we needed to add it again but yeah there's always git history.

Reply:
Also the `resource :questions, only: []` doesn't do anything so I'll take it out too.